src/csvcubed/utils/sparql_handler/sparqlmanager.py

```diff
@@ -367,6 +373,26 @@ def select_codelist_cols_by_dataset_url(
         )
     return map_codelist_cols_by_dataset_url_result(results)
 
+def select_primary_key_col_names_by_dataset_url(
```

Comment:
Can you replace `dataset_url` with `csv_url` here? I believe that's what we're matching on.